pkg/configtx/application.go

```diff
@@ -234,9 +234,9 @@ func (c *ConfigTx) AnchorPeers(orgName string) ([]Address, error) {
 	return anchorPeers, nil
 }
 
-// AddApplicationOrg adds an organization to an existing Application configuration.
-// Will not error if organization already exists.
-func (c *ConfigTx) AddApplicationOrg(org Organization) error {
+// SetApplicationOrg adds an organization to an existing Application configuration.
```

Comment:
We could be more explicit about what this is actually doing. Something like `SetApplicationOrg sets the organization config group for the given application org key in an existing Application configuration's Groups map.`

Reply:
done